sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -426,7 +426,6 @@ private[hive] class HiveClientImpl(
         // in the function toHiveTable.
         properties = properties.filter(kv => kv._1 != "comment" && kv._1 != "EXTERNAL"),
         comment = properties.get("comment"),
-        viewOriginalText = Option(h.getViewOriginalText),
         viewText = Option(h.getViewExpandedText),
```

Comment:
Could you add a comment in this line to explain why we are using `h.getViewExpandedText` instead of `h.getViewOriginalText`?